src/test/java/edu/cornell/kfs/gl/service/impl/ScheduledAccountingLineServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+package edu.cornell.kfs.gl.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.fp.CuFPConstants;
+import edu.cornell.kfs.fp.CuFPConstants.ScheduledSourceAccountingLineConstants.ScheduleTypes;
+import edu.cornell.kfs.fp.businessobject.ScheduledSourceAccountingLine;
+import edu.cornell.kfs.fp.service.impl.ScheduledAccountingLineServiceImpl;
+
+public class ScheduledAccountingLineServiceImplTest {
+	
+	ScheduledAccountingLineServiceImpl scheduledAccountingLineUtil;
+	Date startingPointDate;
+	Calendar startingPointCal;
+	ScheduledSourceAccountingLine accountingLine;
+
+	@Before
+	public void setUp() throws Exception {
+		scheduledAccountingLineUtil = new ScheduledAccountingLineServiceImpl();
+		
+		startingPointDate = new Date(Calendar.getInstance().getTimeInMillis());
+		startingPointCal = Calendar.getInstance();
+		startingPointCal.setTimeInMillis(startingPointDate.getTime());
+		
+		accountingLine = new ScheduledSourceAccountingLine();
+		accountingLine.setStartDate(startingPointDate);
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		scheduledAccountingLineUtil = null;
+		startingPointDate = null;
+		startingPointCal = null;
+		accountingLine = null;
+	}
+	
+	private Date getDateResult(ScheduleTypes scheduleType, String count) {
```

Comment:
per Clean Code, I think I would move these two private methods to be after the test methods